README.md

```diff
@@ -45,9 +41,69 @@ Dict{String, Array} with 27 entries:
   "float64_3"   => [0.0, 1.0, 2.0]
 ```
 
-It is also possible to load just header using unexported `load_header` as 
+It can also perform a lazy loading with `SafeTensors.deserialize("model.safetensors")` which `mmap` the file and return a `Dict`-like object:
+```julia
+julia> tensors = SafeTensors.deserialize("test/model.safetensors"; mmap = true #= default to `true`=#);
+
+julia> tensors["float32_35"]
+3×5 mappedarray(ltoh, PermutedDimsArray(reshape(reinterpret(Float32, view(::Vector{UInt8}, 0x0000000000000ef5:0x0000000000000f30)), 5, 3), (2, 1))) with eltype Float32:
+  0.0   1.0   2.0   3.0   4.0
+  5.0   6.0   7.0   8.0   9.0
+ 10.0  11.0  12.0  13.0  14.0
+```
+
+Serialization is also support:
+
 ```julia
-julia> d = SafeTensors.load_header("model.safetensors")
+julia> weights = Dict("W"=>randn(BFloat16, 3, 5), "b"=>rand(BFloat16, 3))
+Dict{String, Array{BFloat16}} with 2 entries:
+  "W" => [0.617188 0.695312 … 0.390625 -2.0; -0.65625 -0.617188 … 0.652344 0.244141; 0.226562 2.70312 … -0.174805 -0.7773…
+  "b" => [0.111816, 0.566406, 0.283203]
+
+julia> f = tempname();
+
+julia> SafeTensors.serialize(f, weights)
+
+julia> loaded["W"] ≈ weights["W"]
+true
```

Comment:
this should be moved down  I guess

Reply:
It's just for showing that the result is correct

Reply:
I miss a line of deserialize, fixed